src/fields/field_fe.hh

```diff
@@ -295,6 +329,15 @@ private:
     std::array<FEItem, 4> fe_item_;
     MixedPtr<FiniteElement> fe_;
 
+    /// Pointer to computational mesh
+    const Mesh *comp_mesh_;
```

Comment:
Isn't computational mesh part of parent FieldAlgo Base or part of DOF handler?

Reply:
comp_mesh_ removed